src/RestSharp.SnakeCaseSerializer.Tests/SnakeCaseSerializationStrategyTests.cs

```diff
@@ -7,11 +7,22 @@ namespace RestSharp.SnakeCaseSerializer.Tests
     [TestFixture]
     public class SnakeCaseSerializationStrategyTests
     {
-        public SnakeCaseSerializationStrategyTests()
+        private SnakeCaseSerializationStrategy strategyUnderTest;
```

Comment:
Naming format for private variables should be `_myVariable`. Although for some reason VS for Mac doesn't seem to like that format 😒